ansible/roles/test/tasks/lag_vm_ports_test.yml

```diff
@@ -0,0 +1,78 @@
+#---------------------------------------------------------
+# --- Part of lagall.yml test ---
+#---------------------------------------------------------
+# Shutdown each LAG member on VMs.
+#
+# The following actions are to be done for each LAG (and each VM):
+#   1.1) Put DOWN EOS interface on VM (simulation of the halt of sending LACP packets).
+#   1.2) Verify all the traffic goes through another LAG member and no packets lost.
+#
+# @ params: login_creds     -   credentials for Arista VM
+# @ params: vm_name
+# @ params: lag_ip          -   LAG interface IP address
+# @ params: lag_member_0    -   First LAG member on VM
+# @ params: lag_member_1    -   Second LAG member on VM
+# @ params: iface_behind_lag_member_0   -   PTF docker iface, connected to LAG VM
+# @ params: iface_behind_lag_member_1   -   PTF docker iface, connected to LAG VM
+# @ params: not_behind_lag_iface        -   PTF docker interface that is connected to non-LAG VM (meaning eth31)
+# @ params: vm_ip       -    VM mgmt IP; taken from lag_link_flap_vars.yml
+# @ params: num_of_pkts -    number of packets to send while PTF test.
+#---------------------------------------------------------
+
+        # --- TEST FIRST LAG MEMBER ---
+- block:
```

Comment:
this test assumes that each lag has two members, need to check if the test bed type is t1-lag or not before running the test.